package.json

```diff
@@ -84,13 +84,11 @@
 					"description": "Traces the communication between VS Code and the language server."
 				},
 				"biome.lspBin": {
-					"type": ["string", "null"],
-					"default": null,
+					"type": "string",
 					"markdownDescription": "The biome lsp server executable. If the path is relative, the workspace folder will be used as base path"
 				},
 				"biome.rename": {
-					"type": ["boolean", "null"],
-					"default": null,
+					"type": "boolean",
```

Comment:
```suggestion
					"type": "boolean",
					"default": false,
```

Reply:
Similar behavior, best if you want declarative, worst if you want less LOC and default

Reply:
I'd rather be explicit here.